mysql/userstat/userstat.go

```diff
@@ -35,6 +35,7 @@ type MysqlStatPerUser struct {
 	TotalConnections      *metrics.Gauge
 	ConcurrentConnections *metrics.Gauge
 	ConnectedTime         *metrics.Gauge
+	CPUTime               *metrics.Gauge
```

Comment:
Are these actually gauges or you prefer to report counters as point-in-time values?